contributor-guide/modules/ROOT/pages/testing/test-policy.adoc

```diff
@@ -0,0 +1,53 @@
+= Test Policy
+:navtitle: Test Policy
+
+There are required and optional (though highly recommended) tests for libraries.
+
+== Required
+
+Boost uses an automatic regression test suite which generates HTML compiler status tables. Boost also uses xref:testing/continuous-integration.adoc[] to ensure these tests are regularly run. Ensure your library testing includes the following:
+
+. Every Boost library should supply one or more suitable test programs to be exercised by the Boost regression test suite. In addition to the usual _compile-link-run_ tests expecting successful completion, _compile-only_ or _compile-and-link-only_ tests may be performed, and success for the test may be defined as failure of the steps.
+
+. Test program execution must report errors by returning a non-zero value. They may also write to `stdout` or `stderr`, but that output should be relatively brief. Regardless of other output, a non-zero return value is the only way the regression test framework will recognize an error has occurred. Note that test programs to be included in the status tables must compile, link, and run quickly since the tests are executed many times.
+
+. Libraries with time consuming tests should be divided into a fast-execution basic test program for the status tables, and a separate full-coverage test program for exhaustive test cases. The basic test should concentrate on compilation issues so that the status tables accurately reflect the library's likelihood of correct compilation on a platform.
+
+. If for any reason the usual test policies do not apply to a particular library, an alternate test strategy must be described and implemented.
+
+. Authors should supply a Jamfile to drive the regression tests for the library.
+
+== Optional
+
+* Use boost:test[] to create individual test cases and group them into test suites. Then use the library's test tools to verify the expected behavior.
+
+* For simple cases, you can use `lightweight_test.hpp`, instead of boost:test[]. This header file is located in the `<boost-root>/boost/core` directory of your local installation.
```

Comment:
Isn't is easier to mention `boost:core[]`?

Reply:
agreed, I'll reword this